### PR TITLE
Update set-permissions.md

### PR DESCRIPTION
**Pull request by edgargarciamacrometa**
Added the complete example for set permissions

### CODE REVIEW

File: docs/sdks/set-permissions.md [hunk 1]
```diff
@@ -18,44 +18,92 @@ Set permissions for various assets.
...
-var coll = await client.getCollections();
-var collectionName = 'testCollection'
-var streamName = 'testStream'
-console.log("Existing Collections: ", coll.result)
-try{
-    await client.createCollection(collectionName);
-    console.log("Collection Created Successfully")
+const jsc8 = require("jsc8");
+
+// Email and Password to Authenticate client instance
```
Comment (diana-macrometa) [inline docs/sdks/set-permissions.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/sdks/set-permissions.md [hunk 2]
```diff
@@ -18,44 +18,92 @@ Set permissions for various assets.
...
+const fabric = "_system";
+const keyid = "id1";
+const collectionName = "testCollection";
+const streamName = "testStream";
+
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+// Create an authenticated instance with a JWT token.
```
Comment (diana-macrometa) [inline docs/sdks/set-permissions.md:37]:
```suggestion
// Or use one of the following authentication methods.

// Create an authenticated instance with a JWT token.
```

File: docs/sdks/set-permissions.md [hunk 3]
```diff
@@ -18,44 +18,92 @@ Set permissions for various assets.
...
-}
-catch(e){
-    console.log("Failed to set Database Access Level: ",e)
+async function main () {
+  await client
+    .login(email, password)
+    .then((e) => console.log("1. User authentication done!"))
+    .catch((error) => error);
+
+  console.log("\n2. Listing Collections");
```
Comment (diana-macrometa) [inline docs/sdks/set-permissions.md]:
```suggestion
  console.log("\n2. Listing collections");
```

File: docs/sdks/set-permissions.md [hunk 4]
```diff
@@ -18,44 +18,92 @@ Set permissions for various assets.
...
+  await client
+    .getCollections()
+    .then((collections) => {
+      console.log(collections);
+    })
+    .catch((error) => messageHandler(error));
+
+  console.log("\n3. Creating Collection");
```
Comment (diana-macrometa) [inline docs/sdks/set-permissions.md]:
```suggestion
  console.log("\n3. Creating collection");
```

File: docs/sdks/set-permissions.md [hunk 5]
```diff
@@ -18,44 +18,92 @@ Set permissions for various assets.
...
+  await client
+    .createCollection(collectionName)
+    .then((collection) => {
+      console.log("Collection Created Successfully");
```
Comment (diana-macrometa) [inline docs/sdks/set-permissions.md]:
```suggestion
      console.log("Collection created successfully");
```

File: docs/sdks/set-permissions.md [hunk 6]
```diff
@@ -18,44 +18,92 @@ Set permissions for various assets.
...
+
+  console.log("\n4. Setting collection access level access level to read write");
+  await client
+    .setCollectionAccessLevel(keyid, fabric, collectionName, "rw")
+    .then((collectionAccessLevel) => {
+      console.log(collectionAccessLevel);
+    })
+    .catch((error) => messageHandler(error));
+
+  console.log("\n5. Creating Stream " + streamName);
```
Comment (diana-macrometa) [inline docs/sdks/set-permissions.md]:
```suggestion
  console.log("\n5. Creating stream " + streamName);
```

File: docs/sdks/set-permissions.md [hunk 7]
```diff
@@ -18,44 +18,92 @@ Set permissions for various assets.
...
+  await client
+    .createStream(streamName)
+    .then((stream) => console.log(stream))
+    .catch((error) => messageHandler(error));
+
+  console.log(
+    "\n6. Setting Stream " + streamName + " access level to read only"
```
Comment (diana-macrometa) [inline docs/sdks/set-permissions.md]:
```suggestion
    "\n6. Setting stream " + streamName + " access level to read only"
```

File: docs/sdks/set-permissions.md [hunk 8]
```diff
@@ -18,44 +18,92 @@ Set permissions for various assets.
...
+  );
+  await client
+    .setStreamAccessLevel(keyid, fabric, "c8globals." + streamName, "ro")
+    .then((streamAccessLevel) => console.log(streamAccessLevel))
+    .catch((error) => messageHandler(error));
+
+  console.log(
+    "\n7. Setting Database " +
```
Comment (diana-macrometa) [inline docs/sdks/set-permissions.md]:
```suggestion
    "\n7. Setting database " +
```

**Review by diana-macrometa — APPROVED** [review]
A few minor suggestions, then g2g.

File: docs/sdks/set-permissions.md [hunk 9]
```diff
@@ -18,44 +18,94 @@ Set permissions for various assets.
...
+
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+// Or use one of the following authentication methods.
+
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/sdks/set-permissions.md]:
```suggestion
// Create an authenticated instance with an API key.
```

**Review by jamesm-macrometa — APPROVED** [review]
Looks good!

**Pull request merged**